commons/commons-test/src/main/kotlin/ebi/ac/uk/test/TemporaryFolderExt.kt

```diff
@@ -23,6 +23,15 @@ fun TemporaryFolder.createOrReplaceFile(fileName: String): File {
     return createFile(fileName)
 }
 
+/**
+ * Creates a file with the given name and content or replaces it if already exist.
+ */
+fun TemporaryFolder.createOrReplaceFile(fileName: String, content: String): File {
```

Comment:
Couldn't this lead to failures? Maybe it'd be better just cleaning the temp folder after each test if required.